types/splitio.d.ts

```diff
@@ -242,6 +243,16 @@ interface ISharedSettings {
      * @property {Object} localhostMode
      */
     localhostMode?: SplitIO.LocalhostFactory
+    /**
+     * Controls the SDK continuous synchronization flags. 
+     * 
+     * When `true` a running SDK will process rollout plan updates performed on the UI (default). 
+     * When false it'll just fetch all data upon init
+     * 
+     * @property {boolean} enabled
+     * @default true
+     */
+    enabled?: boolean
```

Comment:
The ISharedSettings could disappear entirely, we'll move all of this to IBrowserBasicSettings.